release/rancher/rancher_test.go

```diff
@@ -0,0 +1,41 @@
+package rancher
```

Comment:
This file also needs to be formatted. This makes me think that the code also needs to go through a known-good vet lint as well since the formatted isn't working.